.github/workflows/deploy.yml

```diff
@@ -0,0 +1,29 @@
+name: Deploy
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
+      - name: Install and configure poetry
+        run: python -m pip install poetry
+      - name: Install python packages
+        run: python -m poetry install
+      - name: Bump files
+        run: python -m poetry run python run-ci.py deploy --git-tag "$(git describe --tags)"
```

Comment:
fells weird to call the command `deploy` before building the package. Why not calling it `bump` or `bump-version`?

Reply:
right!